x-pack/plugins/triggers_actions_ui/public/application/components/rules_setting/query_delay/rules_settings_query_delay_section.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import {
+  MAX_QUERY_DELAY,
+  MIN_QUERY_DELAY,
+  RulesSettingsQueryDelayProperties,
+} from '@kbn/alerting-plugin/common';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiSpacer,
+  EuiText,
+  EuiEmptyPrompt,
+  EuiTitle,
+} from '@elastic/eui';
+import { RulesSettingsRange } from '../rules_settings_range';
+
+const queryDelayDescription = i18n.translate(
+  'xpack.triggersActionsUI.rulesSettings.modal.queryDelayDescription',
+  {
+    defaultMessage:
+      'Increase the query time window to mitigate gaps in data availability due to the ES index refresh interval.',
```

Comment:
I think it would be useful to get some copy review here, not because the sentence is wrong I just think it is carrying a lot of weight. Might just be me, but I feel like I'd need a diagram to understand how this interacts with other alert settings like the alert interval.

Reply:
Yes definitely! @lcawl can you help here 🙂